lib/js/dom.nim

```diff
@@ -39,6 +39,9 @@ type
     onselect*: proc (event: Event) {.nimcall.}
     onsubmit*: proc (event: Event) {.nimcall.}
     onunload*: proc (event: Event) {.nimcall.}
+    onloadstart*: proc (event: Event) {.nimcall.}
```

Comment:
well now this can work inside a ``when``.

Reply:
ok, let's try